cli/src/pcluster/templates/cluster_stack.py

```diff
@@ -1247,6 +1247,7 @@ def _add_head_node(self):
                     "base_os": self.config.image.os,
                     "region": self.stack.region,
                     "shared_storage_type": self.config.head_node.shared_storage_type.lower(),
+                    "default_user_home": self.config.head_node.default_user_home.lower(),
```

Comment:
Shouldn't this setting be reflected also in compute and login nodes?
I'm thinking about the case where the head node has `DefaultUserHome=Local`. Without reflecting the setting to compute/login nodes, these nodes would have the default user home directory in the usual /home whereas head node would have it in /local/home. Aren't there any side effect of this misalignment?

Reply:
Maybe it needs to be moved to a different section.  It should apply to all instance types in the cookbook already

Reply:
Moved to `DeploymentSettings`

Reply:
Thanks for moving it to the more general `DeploymentSettings`.
I'm still not getting why we add this property only in the head node dna.json. If this applies to all cluster nodes it should be applied to every dna.json. How could it be applied to every cluster node if it is only defined within the head node dna.json?

On this topic, I'd agree that we should have a common dna.json where we define common properties just once, but since we do not have it atm i guess we should replicate the property in every dna.json.

Reply:
added to both queues and login nodes stacks!